java/src/test/java/com/powsybl/python/SeriesArrayHelperTest.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.python;
+
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.test.EurostagTutorialExample1Factory;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * @author Yichen TANG <yichen.tang at rte-france.com>
+ */
+class SeriesArrayHelperTest {
+
+    @Test
+    void testGetter() {
+        Network network = EurostagTutorialExample1Factory.create();
+
+        final SeriesPointerArrayBuilder builder = SeriesArrayHelper.prepareData(network, PyPowsyblApiHeader.ElementType.BUS);
+
+        assertEquals("VLGEN_0", ((SeriesPointerArrayBuilder.StringSeries) builder.seriesList.get(0)).stringGetter.apply(builder.getElements().get(0)));
```

Comment:
For tests it would be better to not need to apply all that logic, but to get arrays, like on python side, in which we can check the values.
Actually, getting tables from the network could be an interesting feature for java too, independently of C interface.

Maybe in a simple approach we can just add a method to the builder to get data as java arrays (+ names) ?